package.json

```diff
@@ -28,8 +28,8 @@
     "babel-traverse": "^6.7.6",
     "babel-types": "^6.7.2",
     "babylon": "^6.7.0",
+    "better-sqlite3": "^4.0.2",
     "lodash": "^4.6.1",
-    "mkdirp": "^0.5.1",
     "pify": "^2.3.0",
```

Comment:
Was this going to go do devDeps?

Reply:
Yep! It was a `dependency` before when the cache used it to make a `cacheDir`. Now it's only used in the smoke tests.